tokens/0xfedae5642668f8636a11987ff386bfd215f942ee.yaml

```diff
@@ -0,0 +1,21 @@
+---
+__FORKDELTA_CUSTOM_SYMBOL: AVOID CONFLICT WITH 0x562952c749d05dca4cd004489a153c7ee7e58240
```

Comment:
It appears as though [PallyCoin PAL by pally.co](https://github.com/forkdelta/tokenbase/blob/master/tokens/0x562952c749d05dca4cd004489a153c7ee7e58240.yaml) is already assigned PAL symbol. We need to figure out a workable arrangement with policypal.network and pally.co folks to resolve this conflict. The custom assigned ticker may go into the `__FORKDELTA_CUSTOM_SYMBOL` field here.

Reply:
OK. does this mean we wait for this, before the merge?

Reply:
@therealmikkok well, we need to assign it a unique symbol/ticker. We can either do it on our own, or offer the parties (and policypal.network peeps specially) a chance to pick their own. Can we reach out to them?